governance/votes/Executive vote - November 28, 2024.md

```diff
@@ -0,0 +1,272 @@
+| JuliaChang | 4,000 | [0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7](https://etherscan.io/address/0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7) |
+| Vigilant | 4,000 | [0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61) |
+
+### Atlas Core Development Payment to Key Atlas Contributors
+
+- **Authorization**: [Ecosystem Authorization](https://forum.sky.money/t/atlas-core-development-payment-requests-november-2024/25580/8), Atlas Axis Authorizations [1](https://forum.sky.money/t/atlas-core-development-payment-requests-november-2024/25580/4), [2](https://forum.sky.money/t/atlas-core-development-payment-requests-november-2024/25580/6)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/atlas-core-development-payment-requests-november-2024/25580)
+
+If this executive proposal passes, then a total of **90,277 USDS** and **330,000 SKY** will be distributed to three Atlas Core Development key contributors.
+
+| Contributor | Amount | Address |
+|---|---|---|
+| Kohla (Cloaky) | 20,000 USDS | [0x73dFC091Ad77c03F2809204fCF03C0b9dccf8c7a](https://etherscan.io/address/0x73dFC091Ad77c03F2809204fCF03C0b9dccf8c7a) |
+| Ennoia (Cloaky) | 20,110 USDS | [0xA7364a1738D0bB7D1911318Ca3FB3779A8A58D7b](https://etherscan.io/address/0xA7364a1738D0bB7D1911318Ca3FB3779A8A58D7b) |
+| BLUE | 50,167 USDS | [0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf](https://etherscan.io/address/0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf) |
+| BLUE | 330,000 SKY | [0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf](https://etherscan.io/address/0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf) |
+
+### Spark Proxy Spell
+
+- **Authorization**: [Ecosystem Authorization](https://forum.sky.money/t/28-nov-2024-proposed-changes-to-spark-for-upcoming-spell/25543/3), [Ecosystem Authorization](https://forum.sky.money/t/28-nov-2024-proposed-changes-to-spark-for-upcoming-spell-amendments/25575/3), [Ecosystem Authorization](https://forum.sky.money/t/stability-scope-parameter-changes-18-sfs-dsr-ssr-spark-effective-dai-borrow-rate-spark-liquidity-layer/25593/2), Governance Polls [1](https://vote.makerdao.com/polling/QmSxJJ6Z), [2](https://vote.makerdao.com/polling/QmaxFZfF), [3](https://vote.makerdao.com/polling/QmWUkstV), [4](https://vote.makerdao.com/polling/QmQ2Umfm), [5](https://vote.makerdao.com/polling/QmcNd4mH)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/28-nov-2024-proposed-changes-to-spark-for-upcoming-spell/25543), [Forum Post](https://forum.sky.money/t/28-nov-2024-proposed-changes-to-spark-for-upcoming-spell-amendments/25575)
+
+If this executive proposal passes, then a Spark Proxy Spell at [0x6c87D984689CeD0bB367A58722aC74013F82267d](https://etherscan.io/address/0x6c87D984689CeD0bB367A58722aC74013F82267d) will be executed. The expected contents are as follows:
+
+#### [Mainnet] Spark Effective DAI Borrow Rate Increase
+
+- Increase the Spark Effective DAI Borrow Rate by ~1%, from ~8.5% to **~9.5%**.
```

Comment:
I changed this because I thought it was more consistent to keep the same units throughout. Why do you think using percentage points instead would be better?

Reply:
I see the other sections follow your approach. I think it’s better to adjust my part rather than changing the entire document